packages/nextjs/components/scaffold-eth/Input/IntegerInput.tsx

```diff
@@ -46,7 +46,7 @@ export const IntegerInput = ({
         !inputError &&
         !disableMultiplyBy1e18 && (
           <div
-            className="space-x-4 flex tooltip tooltip-top tooltip-secondary before:content-[attr(data-tip)] before:right-[-10px] before:left-auto before:transform-none"
+            className="space-x-4 flex tooltip tooltip-top tooltip-info before:shadow-lg before:content-[attr(data-tip)] before:right-[-10px] before:left-auto before:transform-none"
```

Comment:
In light mode tooltip tail is hidden(kind of present in prod version too): 

![Screenshot 2024-06-27 at 9 52 53 PM](https://github.com/BuidlGuidl/abi.ninja/assets/80153681/70e97983-3feb-4d40-8663-93624c777e76)

Also on darkmode white tooltip kindof seems too much pop out

---

I was wondering instead of using `tooltip-info`  here what if we update `tailwind.config.js` (since we already have overriden dark mode config) to something like this, in both light and dark mode?:  

![image](https://github.com/BuidlGuidl/abi.ninja/assets/80153681/78dfe977-a6b9-45c8-b02b-ecb9e7db03ef)


Using secondary and secondary-content this is how it looks on dark mode and light mode: 

| light | Dark | 
| ------- | ----- | 
| ![image](https://github.com/BuidlGuidl/abi.ninja/assets/80153681/a8b4bf57-7f10-4830-878c-71a02ec89d8b) |  ![image](https://github.com/BuidlGuidl/abi.ninja/assets/80153681/2f30bc6c-1556-41bb-86b5-da483ef3b198) | 

But we could play around with other colors as well primary etc, you could check their respective

Reply:
That looks way better than what I suggested! Thanks! Committing the changes now